src/EFCore.PG/Storage/Internal/Mapping/NpgsqlArrayTypeMapping.cs

```diff
@@ -39,32 +40,39 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.Mapping
     /// </remarks>
     public class NpgsqlArrayTypeMapping : RelationalTypeMapping
     {
-        public RelationalTypeMapping ElementMapping { get; }
+        /// <summary>
+        /// The CLR type of the array items.
+        /// </summary>
+        [NotNull] readonly RelationalTypeMapping _elementMapping;
```

Comment:
See @YohDeadfall's comment below, it's probably better to leave this a publicly-accessible property.